src/frontend/citizen-portal/src/app/components/dispute-ticket-summary/dispute-ticket-summary.component.html

```diff
@@ -4,6 +4,75 @@
   </section>
 
   <div class="mb-2">
+    <section class="px-4 py-2 mb-4 card-section">
+      <div class="page-subheader pagebreak">
+        <strong> Disputant contact information </strong>
+          <div class="row mb-2">
+            <div class="lg-2 col-lg-4">
+              <small class="d-block">Surname</small>
+              <div class="text"><strong>{{ noticeOfDispute.surname | titlecase }}</strong></div>
+            </div>
+            <div class="col-lg-4">
+              <small class="d-block">Given names</small>
+              <div class="text"><strong>{{ noticeOfDispute.given_names | titlecase }}</strong></div>
+            </div>
+          </div>
+
+          <div class="row mb-2">
+            <div class="lg-2 col-lg-4">
+              <small class="d-block">Mailing address</small>
+              <div class="text"><strong>{{ noticeOfDispute.address | titlecase }}</strong></div>
+            </div>
+            <div class="col-lg-4">
+              <small class="d-block">Country</small>
+              <div class="text"><strong>Canada</strong></div>
```

Comment:
{{ noticeOfDispute.country }} should be hard coded to "Canada" now. Please use it. Thanks

Reply:
From what I can see, the noticeOfDispute object doesn't contain the country attribute.  The ticketFormFields object does, but it is not passed into the dispute-ticket-summary component.

Reply:
Oh yes, the noticeOfDispute object do not have country attribute, so it will cause error.
But you can try {{ noticeOfDispute.["country"] }} because the value should be there as JS/TS will not drop the attributes.

Reply:
Yeah it was giving a compilation error, so I've just removed it completely for now, since it doesn't really make sense to show the country, if we are only supporting Canada for now.